pages/_document.js

```diff
@@ -10,11 +10,10 @@ class MyDocument extends Document {
     return (
       <Html>
         <Head>
-          <meta charset="UTF-8" />
+          <meta charSet="UTF-8" />
```

Comment:
Why is this camelcased?
I thought previous was fine: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta

Reply:
Its the difference b/w HTML and react. React has camel-cased everything.

Reply:
Thought so. Ok, makes sense. Thanks